src/test/java/org/folio/moduserstest/RestVerticleIT.java

```diff
@@ -1226,7 +1227,11 @@ public void test1Sequential(TestContext context) {
       .compose(v -> findAndDeleteProxyfor(context))
       .compose(v -> createTestDeleteObjectById(context, testAddress, "/addresstypes", true))
       .compose(v -> createTestDeleteObjectById(context, testGroup, "/groups", true))
-      .compose(v -> createTestDeleteObjectById(context, testProxyFor, "/proxiesfor", true));
+      .compose(v -> createTestDeleteObjectById(context, testProxyFor, "/proxiesfor", true))
+      .compose(v -> getPatronBlockConditions(context))
+      .compose(v -> getPatronBlockCondition(context, MAX_NUMBER_OF_LOST_ITEMS_CONDITION_ID))
+      .compose(v -> putPatronBlockCondition(context, MAX_NUMBER_OF_LOST_ITEMS_CONDITION_ID))
+      .compose(v -> deletePatronBlockCondition(context, MAX_NUMBER_OF_LOST_ITEMS_CONDITION_ID));
```

Comment:
Can you move the new tests into a dedicated test class in https://github.com/folio-org/mod-users/tree/master/src/test/java/org/folio/rest/impl ?

Reply:
moved